checkstyle.xml

```diff
@@ -34,7 +34,9 @@
 
     <module name="SuppressWarningsFilter"/>
 
-    <!--<module name="NewlineAtEndOfFile" />--> <!-- Screwed up on Appveyor for some reason -->
+    <module name="NewlineAtEndOfFile">
+        <property name="lineSeparator" value="lf"/>
```

Comment:
Here is the related JavaDoc: http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/NewlineAtEndOfFileCheck.html